Library/Homebrew/extend/os/pathname.rb

```diff
@@ -0,0 +1 @@
+require "extend/os/linux/extend/pathname" if OS.linux?
```

Comment:
The equivalent file on macOS is `os/mac/pathname.rb`. That file should probably be moved to `extend/os/mac/extend/pathname.rb`

Reply:
done

Reply:
`if/`elsif`